test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,333 @@
+-   Check if the `Separator` is render outside before/after the `Group`
+
+Expected look:  
+![SeparatorRootBefore](../resources/group-separator-before-paragraph.png)
+
+--------------------------------------------------------------------------------
+
+##### TC007
+
+### Insertion point (Separator) is show in correct place in Group
+
+-   Add a `Group` block
+-   Press group `AppenderButton ( + )`
+-   Check if the `Separator` is render instead of `AppenderButton`
+
+Expected look:  
+![InnerSeparatorRoot](../resources/group-inner-separator-root.png)
+
+-   Add a `Group` block
+-   Add block inside
+-   Select `Group` block
+-   Press group `AppenderButton ( + )`
+-   Check if the `Separator` is render after last block in the `Group`
+
+Expected look:  
+![InnerSeparatorAfter](../resources/group-inner-separator-after.png)
+
+-   Add a `Group` block
+-   Add block inside and keep newly added block selected
+-   Press `AppenderButton ( + )`
+-   Select where you want to add block after or before
+-   Check if the `Separator` is render after/before selected block in the `Group`
+
+Expected look:  
+![InnerSeparatorBefore](../resources/group-inner-separator-before.png)
+
+--------------------------------------------------------------------------------
+
+##### TC008
+
+### Nested block have proper border styling
+
+-   Add a `Group` block
+-   Nest some blocks inside
+-   Check if each selected block gets solid blue border
+-   Check if each first child gets dashed grey border
+-   Check if margins and paddings do not bounce when changing selection (exception is the trasition of the selected block connected with making space for FloatingToolbar)
+-   Check if there is no weird/unexpected extra free space around blocks
+
+Expected look:  
+![BorderStyling](../resources/group-border-styling.png)
+
+--------------------------------------------------------------------------------
+
+##### TC009
+
+### Nested empty group block do not multiplicates margins
+
+-   Add a `Group` block
+-   Nest multiple empty `Group` block
+-   Check if there is no extra margins applied when nested empty `Group` blocks
+
+Expected look:  
+![PlaceholderNested](../resources/group-placeholder-nest-1.png)
+
+--------------------------------------------------------------------------------
+
+##### TC010
+
+### Nested selection cause applying dimmed style on the rest of blocks
+
+-   Add a `Group` block
+-   Create some nested structure
+-   Navigate down in the hierarchy
+-   Check if each time you change selection all blocks that are higher in hierarchy tree gets dimmed
+-   Select `Group` block
+-   Check if none block gets dimmed
+
+Expected look:  
+![Dimming1](../resources/group-dimming-1.png)
+  
+![Dimming2](../resources/group-dimming-2.png)
+  
+![Dimming3](../resources/group-dimming-3.png)
+
+--------------------------------------------------------------------------------
+
+##### TC011
+
+### No extra margins applied when nesting empty group blocks
```

Comment:
title and steps don't match

Reply:
I removed it because it should check if diming state isn't remember when you close the post. However I realise that we do not had this issue on production but only on demo app.